ports/atmel-samd/boards/monster_m4sk/mpconfigboard.h

```diff
@@ -0,0 +1,31 @@
+#define MICROPY_HW_BOARD_NAME "Adafruit Monster M4SK"
+#define MICROPY_HW_MCU_NAME "samd51j19"
+
+#define CIRCUITPY_MCU_FAMILY samd51
+
+#define MICROPY_HW_LED_STATUS   (&pin_PA27)
+
+// These are pins not to reset.
+// QSPI Data pins
+#define MICROPY_PORT_A (PORT_PA08 | PORT_PA09 | PORT_PA10 | PORT_PA11)
+// DotStar pins, QSPI CS, and QSPI SCK
+#define MICROPY_PORT_B (PORT_PB10 | PORT_PB11)
+#define MICROPY_PORT_C (0)
+#define MICROPY_PORT_D (0)
+
+#define AUTORESET_DELAY_MS 500
+
+// If you change this, then make sure to update the linker scripts as well to
+// make sure you don't overwrite code
+#define CIRCUITPY_INTERNAL_NVM_SIZE 8192
+
+#define BOARD_FLASH_SIZE (FLASH_SIZE - 0x4000 - CIRCUITPY_INTERNAL_NVM_SIZE)
+
+#define DEFAULT_I2C_BUS_SCL (&pin_PA01)
+#define DEFAULT_I2C_BUS_SDA (&pin_PA00)
+
+// USB is always used internally so skip the pin objects for it.
+#define IGNORE_PIN_PA24     1
+#define IGNORE_PIN_PA25     1
+
+#define CIRCUITPY_FS_NAME "MONSTERM4SK"
```

Comment:
Should I remove it or file an issue on Mu? I wanted to swap it because the current M4SK instructions use CIRCUITPY to init the drive and then wipe out CircuitPython with eyeball code.

Reply:
I think it would be better to remove it and just leave it as CIRCUITPY. Mu depends on that name a lot of places, and all our documentation assumes it as well. I'd see this as a constant support issue. Using CircuitPython to init the M4SK is a hack, and can be improved in the future. In either case, there's going to be some confusion about the filesystem name; I think it would be easier to explain the M4SK drive as `CIRCUITPY` when not using CPy than as MONSTERM4SK when using CPy.

Reply:
Ok, sounds good.